### PR TITLE
More stylesheet cleanup

### PR DESCRIPTION
**Pull request by xtine**
* Some more minor stylesheet cleanup
* Remove `grunt-contrib-watch` as it is not being used

### CODE REVIEW

**Comment by cmc333333** [comment]
Travis is erroring on
```
Error: Undefined mixin 'regulation-nav-item-font'.
```

**Comment by xtine** [comment]
@cmc333333: fixed!

File: Gruntfile.js [hunk 1]
```diff
@@ -48,25 +48,6 @@ module.exports = function(grunt) {
...
-     */
-    less: {
-        dev: {
-            options: {
-                paths: ['<%= env.frontEndPath %>/css/less'],
-                compress: false,
-                sourceMap: true,
-                sourceMapFilename: '<%= env.frontEndPath %>/css/style.css.map',
-                sourceMapBasepath: '<%= env.frontEndPath %>/css/less/',
-                sourceMapURL: 'style.css.map'
```
Comment (cmc333333) [inline Gruntfile.js]: Curious: are we getting sourcemaps on the SASS?

**Comment by cmc333333** [comment]
Looks good!

**Pull request merged**